news/ppc64le-on-travis.rst

```diff
@@ -0,0 +1,6 @@
+**Added:**
+
+* Added native ppc64le support to for travis-ci.  This can be enabled by either using
+  `provider: {linux_ppc64le: travis}` or `provider: {linux_ppc64le: native}` in the conda-forge.yml.
+  These should only be used when they are required as many recipes build just fine under emulation using
+  qemu on azure which will remain the default behavior.
```

Comment:
Now that we don't build on travis-ci anymore, what do you think about making travis the default for ppc64le?

Reply:
given the low amount of build capacity on travis, i'd still prefer just using qemu where feasible (which seems to be for about 95% of feedstocks)

Reply:
I don't think that should be an issue because IBM is providing build machines to travis and we could talk with @jayfurmanek if the queue becomes too long. Also running on native hardware is faster and the 6 hour build limit of azure is not much use though. (For eg: numpy 9 mins vs 58 mins for native vs qemu)

Reply:
@conda-forge/core, thoughts?

Reply:
If the queue does not get as bad as OS X I guess that building on native hardware is always a plus.

Reply:
Would it be possible to do a trial run on native and see what the backlog looks like?

Reply:
There's only a few packages that we are building on ppc64le, so can't see any queue.

Reply:
Hmm, should we put out a broader call? It might be helpful to get input from potential users as to which packages they'd want to see built. (My understanding is this mostly a problem of scope, small scope favors travis, large azure)

Reply:
Running on native hardware is always preferred as it takes one step out of the debugging process if a build fails (emulation is always a suspect) and is faster.  I would vote to run on Travis by default and switch back if the queue gets too long.

Reply:
I think this is worth it.  I agree that running on native hardware is preferred.